StalkStocks/StalkStocks
# Fetches stock data from API, renders to charts

Description
This feature successfully pulls down stock data from the alphaVantage API.  It populates the data on a graph using Chart.js.

Asana Task
N/A

Testing Instructions
Install all dependancies.  Get private API key to test stock data querying.  Navigate to localhost:3000 to see populated data fields.  Alternatively, send get request to localhost:3000/api to check that the response body is a JSON object with appropriate stock data.

Checklist

Additional Notes, Images, etc.
N/A
